Section 3.1.1 (Federation) copyedits (#28)
rfc-editor/draft-nottingham-avoiding-internet-centralization: draft-nottingham-avoiding-internet-centralization.md

### Federation {#federation}

> Protocol designers often attempt to address centralization through federation, i.e., designing a function in a way that uses independent instances that maintain connectivity and interoperability to provide a single cohesive service. Federation promises to allow users to choose the instance they associate with and accommodates substitution of one instance for another, lowering switching costs.

Reviewer: "i.e.," is a bit awkward here, can we try a different construction?

Reviewer 2: Would this update be acceptable?

```suggestion
Protocol designers often attempt to address centralization through federation by designing a function in a way that uses independent instances that maintain connectivity and interoperability to provide a single cohesive service. Federation promises to allow users to choose the instance they associate with and accommodates substitution of one instance for another, lowering switching costs.
```

Reviewer: Let's stick with "i.e.".

Reviewer 2: We have left "i.e." as is. @mnot, please review and approve if all other copyedits in this section are acceptable.
